gcp.mk

```diff
@@ -0,0 +1,46 @@
+# run the following from the project root:
+# make -f gcp.mk <make rule>
+
+# gCloud variables cloud hosted k8s testing
+GCP_REGION ?= us-west1
+GCP_PROJECT ?=
+
+# directories for cloud hosted k8s infrastructure for running tests
+TF_GKE_SRC_DIR ?= $(INTEGRATION_TEST_ROOT)/infra/gke
+TF_GKE_STATE_DIR ?= $(TF_GKE_SRC_DIR)/state
+
+include ./Makefile
+
+##@ GKE
+
+.PHONY: create-gke
+create-gke: ## Create a new GKE cluster
+	@mkdir -p $(TF_GKE_STATE_DIR)
+	cp -v $(TF_GKE_SRC_DIR)/*.tf $(TF_GKE_STATE_DIR)/.
+	$(TERRAFORM) -chdir=$(TF_GKE_STATE_DIR) init -upgrade
+	$(TERRAFORM) -chdir=$(TF_GKE_STATE_DIR) apply -auto-approve \
+		-var region=$(GCP_REGION) \
+		-var project_id=$(GCP_PROJECT) || exit 1 \
+	rm -f $(TF_GKE_STATE_DIR)/*.tfvars
+	gcloud container clusters get-credentials $$($(TERRAFORM) -chdir=$(TF_GKE_STATE_DIR) output -raw kubernetes_cluster_name) --region $(GCP_REGION)
+
+.PHONY: import-gcp-vars
+import-gcp-vars:
+-include $(TF_GKE_STATE_DIR)/outputs.env
+
+# Currently only supports amd64
+.PHONY: ci-gar-build-push
```

Comment:
Now that we have all of this logic in a sub Makefile we could we simplify the name here, something like `ci-build-push` or `build-push`

Reply:
ci-build-push sounds legit! Will do 👍🏾